test-app/.eslintrc.js

```diff
@@ -47,10 +47,19 @@ module.exports = {
       },
       extends: ['plugin:n/recommended'],
     },
+    // test files
     {
-      // test files
       files: ['tests/**/*-test.{js,ts}'],
       extends: ['plugin:qunit/recommended'],
     },
+    // ts files
+    {
+      files: ['**/*.ts'],
+      extends: [
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+      ],
+      rules: {},
```

Comment:
Should we use the same rules used for the `.eslintrc.js` of the library here? 

Even if it's the test-app, if we ever wish to expand it, we risk that the quality of its code won't match the one of the lib

Reply:
I hesitated a bit (because I wasn't sure to what extent the tests would be typed). I can totally add the same rules

Reply:
I see. Do you think that having the same rules could negatively affect DX on the tests side for past experiences with QUnit maybe?

Reply:
More for "lack of experiences" in that case 😄 I'd say let's add the exact same set of rules in the test-app. In any case, we could still decide later during the migration to keep it or remove it. What do you think?

Reply:
Yep, I think that having the same one provides the safest path now!